woocommerce/wc-admin
# Show "Help" Activity Panel When Working Single Task List Item

Fixes #4593.

This PR seeks to implement most of the "nice to have" functionality described in #4539 

It does not include:
* Search box (it does include the "WooCommerce Docs" item on all help pages)
* Open a ticket button
* Start live chat button

### Accessibility




### Screenshots

![Screen Shot 2020-07-08 at 4 03 25 PM](https://user-images.githubusercontent.com/63922/86965088-05071880-c135-11ea-8bbd-8eb071da4a13.png)

### Detailed test instructions:

- Ensure the Task List is shown
- Go to WooCommerce > Home
- Verify the "Help" tab is not shown
- Click a Task List item
- Verify only the "Help" tab is shown
- Click "Help"
- Verify the contents are relevant to the current task (use list in issue)
- Repeat for all tasks in issue
- Check conditional help items using issue as reference (e.g. WCS, specific payment gateways)
- Verify tracks event on Help tab open (in issue)
- Verify tracks event on help item click (in issue)



### Changelog Note:



Enhancement: show contextual help menu when working on store setup tasks.
